controller/controller_server/csi_controller_server.py

```diff
@@ -614,20 +708,24 @@ def _join_object_prefix_with_name(self, prefix, name):
 
     def GetPluginCapabilities(self, _, __):
         logger.info("GetPluginCapabilities")
-        types = csi_pb2.PluginCapability.Service.Type
+        service_type = csi_pb2.PluginCapability.Service.Type
+        volume_expansion_type = csi_pb2.PluginCapability.VolumeExpansion.Type
         capabilities = self.__get_identity_config("capabilities")
         capability_list = []
-        for cap in capabilities:
+        service_capability = capabilities.get('Service')
+        volume_expansion_capability = capabilities.get('VolumeExpansion')
+        if service_capability:
             capability_list.append(
                 csi_pb2.PluginCapability(
-                    service=csi_pb2.PluginCapability.Service(type=types.Value(cap))
-                )
-            )
-
+                    service=csi_pb2.PluginCapability.Service(type=service_type.Value(service_capability))))
+        if volume_expansion_capability:
+            capability_list.append(
+                csi_pb2.PluginCapability(
+                    volume_expansion=csi_pb2.PluginCapability.VolumeExpansion(
+                        type=volume_expansion_type.Value(volume_expansion_capability))))
```

Comment:
Isn't the result of it "ONLINE" and not expansion?

Reply:
that is how the response built

Reply:
OK. I see the format is "volume_expansion=ONLINE"